### PR TITLE
Adding plot_graph_layout

### PR DESCRIPTION
**Pull request by Arsilla**
Function added to common.py and to embeddingset.

I also added an option of choosing the distance metric you want to use. I couldn't find any other layouts that allowed for a distance input.

### CODE REVIEW

File: whatlies/common.py [hunk 1]
```diff
@@ -24,7 +32,32 @@ def handle_2d_plot(embedding, kind, color=None, xlabel=None, ylabel=None, show_o
         plt.quiver([0], [0], [embedding.vector[0]], [embedding.vector[1]], color=color,
                    angles='xy', scale_units='xy', scale=1)
         plt.text(embedding.vector[0] + 0.01, embedding.vector[1], name)
-    if kind == "text":
+    if (kind == "text") or annot:
```
Comment (koaning) [inline whatlies/common.py]: i like this touch! but i'll likely change the term to `annotate` later. 

value(clarity) > value(saving some characters)

File: whatlies/language.py [hunk 2]
```diff
@@ -28,16 +28,20 @@ def _input_str_legal(string):
...
-        if start != 0:
-            if end != -1:
-                vec = doc[start:end].vector
-        return Embedding(string, vec)
+        if end == -1:
+            end = start + 1
+        if bert:
+            new_str = string.replace("[", "").replace("]", "")
+            return Embedding(string, self.nlp(new_str)[start].vector)
+        else:
```
Comment (koaning) [inline whatlies/language.py]: technically you don't need the else here.

File: whatlies/language.py [hunk 3]
```diff
@@ -28,16 +28,20 @@ def _input_str_legal(string):
...
     def __getitem__(self, string):
         self._input_str_legal(string)
-        doc = self.nlp(string)
-        vec = doc.vector
+        bert = False
         start, end = 0, -1
         split_string = string.split(" ")
         for idx, word in enumerate(split_string):
             if word[0] == "[":
+                bert = True
```
Comment (koaning) [inline whatlies/language.py]: i wonder why these changes were made. could you clarify?

Comment (koaning) [reply whatlies/language.py]: technically ... `bert` is but a single type of contextualized embedding ... odds are there will be many other models too.

File: whatlies/language.py [hunk 4]
```diff
@@ -28,16 +28,20 @@ def _input_str_legal(string):
...
                 end = idx + 1
-        if start != 0:
-            if end != -1:
-                vec = doc[start:end].vector
-        return Embedding(string, vec)
+        if end == -1:
+            end = start + 1
+        if bert:
+            new_str = string.replace("[", "").replace("]", "")
+            return Embedding(string, self.nlp(new_str)[start].vector)
```
Comment (koaning) [inline whatlies/language.py]: i wonder why you only look at `start` here. 

these two examples should yield a different result; 

```
i am going to a [nice beach]
i am going to a [nice] beach
```

**Comment by koaning** [comment]
@Arsilla thanks for the PR! I have some questions about the changes you've made to `__get_item__`. The rest looks green to merge.

File: whatlies/common.py (hunk 1 above)
Comment (koaning) [reply whatlies/common.py]: You don't have to change this for the PR if you don't feel like it. It is a minor details for this phase of the project.

File: whatlies/language.py (hunk 2 above)
Comment (Arsilla) [reply whatlies/language.py]: wow ok super weird, I made these local changes to fix the bugs from the issue #6. I swore I overwrote those with the fixes you've since made. But I guess something went from in my git history. 
I'll change change it back and push the changes tonight. 
(same goes for the other questions related to language.py)

File: whatlies/common.py (hunk 1 above)
Comment (Arsilla) [reply whatlies/common.py]: I guess I was inspired by the seaborn heatmap (I've used that a lot lately), they use annot :)

File: whatlies/language.py (hunk 2 above)
Comment (koaning) [reply whatlies/language.py]: i figured it might be something like that.

**Comment by koaning** [comment]
LGTM

**Comment by koaning** [comment]
thanks for the PR @Arsilla !
